diesel_derives/src/lib.rs

```diff
@@ -597,6 +597,154 @@ pub fn derive_sql_type(input: TokenStream) -> TokenStream {
 /// #     Ok(())
 /// # }
 /// ```
+///
+/// ## Custom Aggregate Functions
+/// Custom aggregate functions can be created in SQLite by adding the `aggregate`
+/// attribute and implementing the
+/// [SqliteAggregateFunction](../diesel/sqlite/trait.SqliteAggregateFunction.html) trait.
```

Comment:
Again suggestions on better wording:
```suggestion
/// ## Custom Aggregate Functions
/// Custom aggregate functions can be created in SQLite by adding an `#[aggregate]`
/// attribute inside of `sql_function`.  `register_impl` needs to be called on the generated function with 
/// a type implementing the  [SqliteAggregateFunction](../diesel/sqlite/trait.SqliteAggregateFunction.html) trait 
/// as type parameter as shown in the examples below.
```